servers/rendering/renderer_rd/forward_mobile/scene_shader_forward_mobile.cpp

```diff
@@ -175,6 +177,10 @@ void SceneShaderForwardMobile::ShaderData::set_code(const String &p_code) {
 	if (uses_transmittance) {
 		WARN_PRINT_ONCE_ED("Transmittance is only available when using the Forward+ renderer.");
 	}
+
+	if (specular_mode == BaseMaterial3D::SPECULAR_MULTISCATTERING_GGX) {
+		WARN_PRINT_ONCE_ED("MultiscatterGGX is only available when using the Forward+ renderer.");
```

Comment:
Again, this warning won't really help the user. There is nothing they can do about it.